packages/core/database/lib/migrations/index.js

```diff
@@ -7,7 +7,7 @@ const { Umzug } = require('umzug');
 const createStorage = require('./storage');
 
 const wrapTransaction = db => fn => () =>
-  db.getConnection().transaction(trx => Promise.resolve(fn(trx)));
+  db.connection.transaction(trx => Promise.resolve(fn(trx)));
```

Comment:
```suggestion
const wrapTransaction = db => fn => () => {
  return db.connection.transaction(async trx => {
    const schema = db.connection.getSchemaName();
    return schema ? fn(trx.withSchema(schema)) : fn(trx);
  });
};
```

Reply:
This is working fine for `.js` files for failing for `.sql` migration files.

Reply:
what error are you getting ?

Reply:
@alexandrebodin - Below error is coming for `.sql` migration file
```
Migration base.sql (up) failed: Original error: knex.raw is not a function
MigrationError: Migration base.sql (up) failed: Original error: knex.raw is not a function
```
`base.sql` contains below query
```
update tabs_components set field='tabsData' where field='tabs'
```